resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -1,269 +1,436 @@
 @extends('app')
-@section('title', 'Generate OFP')
+@section('title', 'SimBrief Flight Planning')
 
 @section('content')
+
+@if(empty(request()->get('aircraft_id')))
+<div class="row">
+  <div class="col-md-12">
+    <h2>Aircraft Selection</h2>
+  </div>
+</div>
+
+<div class="row">
+  <div class="col-md-6">
+		  <select id="aircraftselection" class="form-control select2" onchange="checkacselection()">
+			<option value="ZZZZZ">Please Select An Aircraft</option>
+				@foreach($subfleets as $subfleet)
+					@foreach($subfleet->aircraft as $ac)
+						<option value="{{ $ac->id }}">[ {{ $ac->icao }} ] {{ $ac->registration }}</option>
+					@endforeach
+				@endforeach
+		  </select>
+  </div>
+  <div class="col-md-6">
+    <a id="mylink" style="visibility: hidden" href="{{ route('frontend.simbrief.generate') }}?flight_id={{ $flight->id }}" class="btn btn-primary">Proceed To Flight Planning</a>
+  </div>
+</div>
+@else
+
+  @php
+    $loadmin = $flight->load_factor - $flight->load_factor_variance ;
+    $loadmax = $flight->load_factor + $flight->load_factor_variance ;
+    if($loadmin < 1) { $loadmin = 1 ; }
+    if($loadmax > 100) { $loadmax = 100 ; }
+  @endphp
+
+  @foreach($aircraft as $acdetails)
+    @php
+      $simbrieftype = $acdetails->icao ;
+      $subflid = $acdetails->subfleet_id ;
+      if($acdetails->icao == 'A20N') { $simbrieftype = 'A320' ; }
```

Comment:
These should all be `===` (strict equals)

Reply:
Done